lib/ansible/modules/cloud/azure/azure_rm_functionapp.py

```diff
@@ -34,6 +34,11 @@
     location:
         description:
             - Valid Azure location. Defaults to location of the resource group.
+    plan: 
+        description:
+            - "It can be resource id of existing app service plan. eg.,
```

Comment:
as a common practice, could you pls add support of resourcegroup+name/name also? it  can be a separate pr.

Reply:
Probably in a new pull request. Will leave it as it is in this one.